packages/common/src/visualizations/helpers/styles/tooltipStyles.ts

```diff
@@ -1,11 +1,10 @@
-import { type MantineTheme } from '@mantine/core';
-import type { CSSProperties } from 'react';
+import { GRAY_0, GRAY_1, GRAY_3, GRAY_7, WHITE } from './themeColors';
 
 /**
  * Helper to convert style object to CSS string
  * We need this because ECharts doesn't support style objects directly
  */
-const stylesToString = (styles: CSSProperties) =>
+const stylesToString = (styles: Record<string, unknown>) =>
```

Comment:
Don't have access to CSS properties here - and adding a package here would probably be overkill. lmk if not